articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -113,6 +113,48 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     config = Configurator(settings=settings)
     ```
 
+## Tracking FastAPI applications
+
+1. The following dependencies are required: 
+    - fastapi
+    - uvicorn
+
+2. Add FastAPI middleware.
+
+3. Set span kind server, `span.span_kind = SpanKind.SERVER`.
+
+```python 
+app = FastAPI()
+logger = logging.getLogger(__name__)
+
```

Comment:
Missing `opencensus` imports as well.

Reply:
I didn't add any imports as I noticed the above code snippets don't have imports. but I agree, it is better to add them.
updated as well.